source/guide/distant-interactions.html.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Distant interactions
+icon: globe
+summary: 'How to interact with people, blogs and articles that are on a different instance'
+---
+
+## Interacting with people from other instances
+
+Due to its federated nature, other people using Plume are not necessary on the same instance as you.
+
+However, you can still interact with them (follow them, like their articles, comment on them…). If you
+try to interact with someone or something while not being on your instance, you will be prompted to either
+log in on the current instance, or to enter your full user name (in the `name@instance` format) to be redirected
+to your own instance (where you are logged in).
+
+## Entering the URL manually
+
+You can also make your own instance discover other blogs or accounts by entering their URL manually.
+
+The URL to enter for distant profiles is `https://your.instance/@/name@other.instance`. For blogs it is
+`https://your.instance/~/name@other.instance` (the only difference is that one uses `@`, while the other
+uses `~`).
```

Comment:
i think, from a usuability point of view, it would be much nicer to have a search-box do this, like mastodon does.

Reply:
I think there is an issue for that on https://github.com/Plume-org/Plume

Reply:
(https://github.com/Plume-org/Plume/issues/517)